gnome-extensions/test-settings.sh

```
@@ -0,0 +1,217 @@
+    # Backup current settings
+    dconf dump /org/gnome/shell/extensions/ > "$backup_dir/extensions-backup.dconf"
+    
+    if [ -f "$backup_dir/extensions-backup.dconf" ]; then
+        success "✓ 設定のバックアップが完了しました: $backup_dir"
+    else
+        error "✗ 設定のバックアップに失敗しました"
+        return 1
+    fi
+    
+    log "設定ファイルのサイズを確認中..."
+    local backup_size=$(wc -l < "$backup_dir/extensions-backup.dconf")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)